recipes/foxi/all/conanfile.py

```diff
@@ -0,0 +1,74 @@
+from conans import ConanFile, CMake, tools
+import glob
+import os
+import shutil
+
+
+class FoxiConan(ConanFile):
+    name = "foxi"
+    description = "ONNXIFI with Facebook Extension."
+    license = "MIT"
+    topics = ("conan", "foxi", "onnxifi")
+    homepage = "https://github.com/houseroad/foxi"
+    url = "https://github.com/conan-io/conan-center-index"
+
+    settings = "os", "arch", "compiler", "build_type"
+    options = {"fPIC": [True, False]}
+    default_options = {"fPIC": True}
```

Comment:
Is it possible to build shared?

Hmmm nvm its not supported https://github.com/houseroad/foxi/blob/bd6feb6d0d3fc903df42b4feb82a602a5fcb1fd5/CMakeLists.txt#L52